dashboard/config/libraries/Outbreak.interpreted.js

```diff
@@ -100,9 +102,106 @@ function setupOutbreak(numMonsters, callback) {
   callback();
 }
 
-function layoutSprites(format) {
-  // TODO: Implement layoutSprites
-  console.log(format);
+function layoutBorder() {
+  var spriteIds = getSpriteIdsInUse();
+  var count = spriteIds.length;
+
+  // Layout monster in each corner first
+  if (count > 0) {
+    jumpTo({id: spriteIds[0]}, {x: MIN_XY, y: MIN_XY});
+  }
+  if (count > 1) {
+    jumpTo({id: spriteIds[1]}, {x: MAX_XY, y: MIN_XY});
+  }
+  if (count > 2) {
+    jumpTo({id: spriteIds[2]}, {x: MIN_XY, y: MAX_XY});
+  }
+  if (count > 3) {
+    jumpTo({id: spriteIds[3]}, {x: MAX_XY, y: MAX_XY});
+  }
+
+  // Fill in the sides with however many monsters are left
+  if (count > 4) {
+    var topCount = Math.ceil((count - 4 - 0) / 4);
+    var rightCount = Math.ceil((count - 4 - 1) / 4);
+    var bottomCount = Math.ceil((count - 4 - 2) / 4);
+    var leftCount = Math.ceil((count - 4 - 3) / 4);
+    var i, x, y, frac;
```

Comment:
do we want/need these variables to be shared throughout the code below? i worry that we might introduce state leakage later. if we used `let` to scope each variable to its relevant `for` block, we would reduce that risk (as long as we can use `let` in interpreted libraries).

we could also remove some of these variables -- the x and y calculations are pretty simple below, so we could set them directly instead of assigning them to variables first (e.g., `{x: MAX_XY}` instead of `x = MAX_XY; {x: x}`)

Reply:
we can't use `let` in the interpreter 😭

Reply:
meh, i should've assumed that. so i suppose our only option would be to remove any shared variables that can be removed (likely just `x` and `y`), which i don't feel super strongly about either way

Reply:
oh, you already did that! i didn't realize my comment was already outdated